src/libs/kata-types/src/config/agent.rs

```diff
@@ -27,15 +27,29 @@ pub struct Agent {
     #[serde(default)]
     pub enable_tracing: bool,
 
-    /// Enable debug console.
-    /// If enabled, user can connect guest OS running inside hypervisor through
-    /// "kata-runtime exec <sandbox-id>" command
+    /// Agent server port
     #[serde(default)]
-    pub debug_console_enabled: bool,
```

Comment:
debug console is intentionally deleted?

Reply:
Thanks for your review, sorry for accidentally deleting it has restored it.